test/core/client_channel/lb_policy/pick_first_test.cc

```diff
@@ -269,32 +281,26 @@ TEST_F(PickFirstTest, WithShuffle) {
       BuildUpdate(kAddresses, MakePickFirstConfig(true)), lb_policy_.get());
   EXPECT_TRUE(status.ok()) << status;
   std::vector<absl::string_view> prev_attempt_connect_order;
-  GetOrderAddressesArePicked(kAddresses, &prev_attempt_connect_order);
-  size_t first_shuffle = 0;
-  for (size_t i = 0; i < kAddresses.size(); ++i) {
-    if (kAddresses[i] != prev_attempt_connect_order[i]) {
-      first_shuffle += 1;
-    }
-  }
   // There is 0.1% chance this check fails by design. Not an assert to prevent
   // flake
-  EXPECT_GT(first_shuffle, 0) << "Addresses were not shuffled";
-  size_t shuffle_count = 0;
-  // 3 attempts to reduce flakes. Probability of a flake is 0.001%
-  for (size_t attempt = 0; attempt < 3; ++attempt) {
+  if (absl::MakeConstSpan(prev_attempt_connect_order) ==
```

Comment:
This block doesn't seem useful.  Please remove.

Reply:
It will log if initial shuffle did not work.

Reply:
This is all changed now.